modules/hosted-che/examples/snip_github-action-yaml-example.adoc

```diff
@@ -0,0 +1,30 @@
+.example.yml
+====
+[source,yaml,subs="+quotes,+attributes,+macros"]
+----
+name: Try in Web IDE example
+
+on:
+  pull_request_target:
+    types: [opened]
```

Comment:
in the usage sample, we use also `types: [opened, synchronize]` 
might be better to use it also for consistency with https://github.com/marketplace/actions/try-in-web-ide

Reply:
Ok, not relevant since there is an extra note below ` Optionally configure the activity types from the `on.pull_request_target.types` field to customize when workflow trigger. Activity types such as `reopened` and `synchronize` can be useful.`